R/ggproto.R

```diff
@@ -118,9 +118,9 @@ fetch_ggproto <- function(x, name) {
       res <- fetch_ggproto(super(), name)
     } else {
       cli::cli_abort(c(
-              "{class(x)[[1]]} was built with an incompatible version of ggproto.",
-        "i" = "Please reinstall the package that provides this extension.
-      "))
+        "{.obj_type_friendly {x}} was built with an incompatible version of ggproto.",
```

Comment:
```suggestion
        "{class(x)[[1]]} was built with an incompatible version of ggproto.",
```
`x` is pretty much guaranteed to be a ggproto class due to where this function is called and for ggproto classes, the 'display name' is the first class.